tsl/test/expected/size_utils.out

```diff
@@ -0,0 +1,41 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- Prepare test data for continuous aggregate size function tests
+CREATE TABLE hypersize(time timestamptz, device int);
+SELECT * FROM create_hypertable('hypersize', 'time');
+NOTICE:  adding not-null constraint to column "time"
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             1 | public      | hypersize  | t
+(1 row)
+
+INSERT INTO hypersize VALUES('2021-02-25', 1);
+-- Test size functions on empty continuous aggregate
+CREATE MATERIALIZED VIEW hypersize_cagg WITH (timescaledb.continuous) AS SELECT time_bucket('1 day', "time") AS bucket, AVG(device) AS value FROM hypersize GROUP BY 1 WITH NO DATA;
+SELECT * FROM hypertable_size('hypersize_cagg');
+ hypertable_size 
+-----------------
+           16384
+(1 row)
+
+SELECT * FROM hypertable_detailed_size('hypersize_cagg') ORDER BY node_name;
+ table_bytes | index_bytes | toast_bytes | total_bytes | node_name 
+-------------+-------------+-------------+-------------+-----------
+           0 |        8192 |        8192 |       16384 | 
+(1 row)
+
+-- Test size functions on non-empty countinuous aggregate
+CALL refresh_continuous_aggregate('hypersize_cagg', NULL, NULL);
+SELECT * FROM hypertable_size('hypersize_cagg');
+ hypertable_size 
+-----------------
+           49152
+(1 row)
+
+SELECT * FROM hypertable_detailed_size('hypersize_cagg') ORDER BY node_name;
+ table_bytes | index_bytes | toast_bytes | total_bytes | node_name 
+-------------+-------------+-------------+-------------+-----------
+        8192 |       24576 |       16384 |       49152 | 
+(1 row)
+
```

Comment:
Another check you can add is to make sure you're getting the same values if you call the function using the cagg or the materialization hypertable, for example:
```sql
fabrizio=# SELECT * FROM hypertable_detailed_size('conditions_summary_1')
EXCEPT
SELECT * FROM hypertable_detailed_size('_timescaledb_internal._materialized_hypertable_2');
 table_bytes | index_bytes | toast_bytes | total_bytes | node_name 
-------------+-------------+-------------+-------------+-----------
(0 rows)
```
First you should discover the materialization hypertable schema and name to register it as a `psql` variable, have a look into this as an example: https://github.com/timescale/timescaledb/blob/main/tsl/test/sql/continuous_aggs.sql.in#L55-L63

Reply:
Done :)